exosphere-shared/templates/add-service/exoservice-ls-mongodb/features/support/world.ls

```diff
@@ -6,7 +6,7 @@ require! {
 World = !->
 
   # Fills in _____modelName_____ ids in the placeholders of the template
-  @fill-in-_____modelName_____-ids = (template, done) ->
+  @fill-in-modelName-ids = (template, done) ->
```

Comment:
You should run the tests after the placeholders have been filled in (i.e. if you were to run `exo-add` with one of these services). Do the tests pass after doing that?